nixos/modules/services/accessibility/speechd.nix

```diff
@@ -17,7 +17,10 @@ in
   options.services.speechd = {
     # FIXME: figure out how to deprecate this EXTREMELY CAREFULLY
     enable = mkEnableOption "speech-dispatcher speech synthesizer daemon" // {
-      default = true;
+      # conservative guess, as in ../misc/graphical-desktop.nix
+      default = config.services.xserver.enable || config.services.displayManager.enable;
+      defaultText = lib.literalExpression
+        "config.services.xserver.enable || config.services.displayManager.enable";
```

Comment:
How about we move this into graphical-desktop.nix instead of doing guess work here?

Also just noticed the same 😂

Reply:
We could do that.  I'm not sure which approach is better.

Reply:
I've added the generic graphic module and it's purpose is to collect all those things, that you only want to turn on on graphical systems, like this setting.

Reply:
OK, moved there and retested.